src/Training.TruckWorld.Clone.Backend/TruckWorld.Persistence/EntityConfigurations/UserConfiguration.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+using TruckWorld.Domain.Entities;
+
+namespace TruckWorld.Persistence.EntityConfigurations;
+
+/// <summary>
+/// Configurations class for the User entity in the database.
+/// </summary>
+public class UserConfiguration : IEntityTypeConfiguration<User>
+{
+    /// <summary>
+    /// Configures the entity properties and their mappings.
+    /// </summary>
+    /// <param name="builder"></param>
+    public void Configure(EntityTypeBuilder<User> builder)
+    {
+        builder.HasKey(userId => userId.Id);
+
+        builder.Property(email => email.EmailAddress).HasMaxLength(100).IsRequired();
+        builder.Property(password => password.Password).HasMaxLength(100).IsRequired();
```

Comment:
1. Change from 'email => email.EmailAddress` to `user => user.EmailAddress, same for the on line 22
2. Update max length as 64, length as power of 2 is best I think
3. Update `Password` property name as `PasswordHash`, recreate the migration